WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m

```diff
@@ -33,6 +33,7 @@
 
 NSString *const WPLegacyEditorNavigationRestorationID = @"WPLegacyEditorNavigationRestorationID";
 NSString *const WPLegacyAbstractPostRestorationKey = @"WPLegacyAbstractPostRestorationKey";
+NSString *const WPAppAnalyticsEditorSourceValueLegacy = @"legacy";
```

Comment:
It'd be nice if we could get away from using straight strings by using an `NS_EXTENSIBLE_STRING_ENUM`. This would take a bit of a refactor, but should help the hope age better in the Swift world. Thoughts?